app/Http/Controllers/SinkApiController.php

```diff
@@ -77,6 +78,17 @@ public function scanLocalFiles(Sink $sink): Response
         ]);
     }
 
+    /**
+     * asd
+     */
+    public function getDoc(Sink $sink): Response
+    {
+        $this->authorize('view', $sink);
```

Comment:
Yess!

Reply:
Yess? OK

Reply:
Added proper documentation block to function